_includes/header.html

```diff
@@ -36,15 +35,15 @@
             class="usa-nav__link {% if page.url contains '/what-is-login/' %}usa-current{% endif %}"
             href="{{ '/what-is-login/' | locale_url }}"
           >
-            <span>{{ site.data.[page.lang].settings.nav.what_is_login_gov }}</span>
+            <span class="usa-nav__link__inner">{{ site.data.[page.lang].settings.nav.what_is_login_gov }}</span>
```

Comment:
This isn't a [valid name for a BEM selector](https://en.bem.info/methodology/naming-convention/#naming-rules), since there should only be at most one double-underscore `__` separating block and element (something like `link-inner` would be fine).

(Not a blocker for initial merge)

Reply:
Filed as [LG-6403](https://cm-jira.usa.gov/browse/LG-6403)